cppcache/integration-test/LibraryCallbacks.cpp

```diff
@@ -29,7 +29,7 @@ void dummyFunc() {}
 }  // namespace test
 
 #define SLEEP(x) std::this_thread::sleep_for(std::chrono::milliseconds(x))
-#define LOG LOGDEBUG
+#define LOG LOG_DEBUG
```

Comment:
Our tests shouldn't be using the NC library logger, there's just no need or justification for it. The test harness should be responsible for all reporting. If the mocks and stated expectations and assertions are inadequate to correctly deduce the nature of a failed test, that indicates the test is inadequate. The test harness provides all the facilities necessary to express expected sequences of operations and additional information. Each expectation and and assertion returns a stream object for logging additional information. For example:

`EXPECT_EQ(foo, bar) << "I BLAME ZOIDBERG!";`

There are other facilities for logging additional information, like the `Gfsh` command output, that will only appear in the case of a failure. The test harness should be quiet, with a clean indication of success, and fail with a complete report. The harness knows how to do it. I recommend you map `LOG` to a no-op, and we begin removing all logging from tests.

Reply:
Agree 100% in principle, but I think this is beyond the scope of the current PR.  To give you an idea of the size of this task, here is the result of "find in files" for `LOG`, whole-word only, so I'd only catch test code that was using some variation of this hokey define:

`Matching lines: 2609    Matching files: 112    Total files searched: 1167`

That's a whooooole lot of logging to go remove from code that is targeted for deprecation.